.github/workflows/build-push-images.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and Push Images
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build_push:
+    if: (github.repository == 'kubevirt/hyperconverged-cluster-operator')
+    name: Build and Push Images
+    runs-on: ubuntu-latest
+    env:
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+      REGISTRY_NAMESPACE: kubevirt
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: Build Applications Images
+        run: make container-build
```

Comment:
I maybe miss something here. As fa as I understand, the default image tag is "latest". I don't think we want that. `latest` should be the latest *released* image. 

I would tag each image twice - a uniqe image tag - and that maybe not needed, and another floating tag like `latest-unstable` or anything else but `latest`.

The unique tag is not mandatory, but it will allow rollbacks.

Reply:
What if we tag it with the latest CSV version (currently 1.4.0) for continuous updates throughout the release?